qiskit/circuit/quantumcircuit.py

```diff
@@ -1130,173 +1130,223 @@ def draw(self, output=None, scale=None, filename=None, style=None,
              interactive=False, plot_barriers=True,
              reverse_bits=False, justify=None, vertical_compression='medium', idle_wires=True,
              with_layout=True, fold=None, ax=None, initial_state=False, cregbundle=True):
-        """Draw the quantum circuit.
+        """Draw a quantum circuit to different formats (set by output parameter):
```

Comment:
The reason for the "the" is because it draws `self`.

```suggestion
        """Draw the quantum circuit. The output parameter allows to choose the drawing format:
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5223/commits/bfb8be30672ba6d81187f57c58a4079026e0bb0e. Slight grammar change.